Add code improvement changes in API classes

BushraAsif opened the pull request.

shahbaz-mehar commented on Api/Others/QueryGiftcard.php:
```diff
@@ -24,12 +24,13 @@
 namespace SDM\Altapay\Api\Others;
 
 use SDM\Altapay\AbstractApi;
-use SDM\Altapay\Request\Giftcard;
```
This should be
use SDM\Altapay\Request\Giftcard;

shahbaz-mehar commented on Api/Others/QueryGiftcard.php:
```diff
@@ -24,12 +24,13 @@
 
 use SDM\Altapay\AbstractApi;
-use SDM\Altapay\Request\Giftcard;
+use SDM\Request\Giftcard;
```
This should be
use SDM\Altapay\Request\Giftcard;

shahbaz-mehar commented on Controller/Customer/Index.php:
```diff
@@ -42,13 +42,6 @@ public function __construct(
-     * @return \Magento\Framework\Controller\ResultInterface|ResponseInterface|void
-     * @throws \Magento\Framework\Exception\NotFoundException
-     */
     public function execute()
```
Why removed DocBlock?

shahbaz-mehar commented on Exceptions/ResponseMessageException.php:
```diff
@@ -25,8 +25,4 @@
-    {
-        $this->message = $message;
-    }
 }
```
Please revert this

shahbaz-mehar commented on Model/Handler/OrderLinesHandler.php:
```diff
@@ -232,7 +232,6 @@ public function sendShipment($order)
             }
         }
-        return false;
     }
```
Please revert this

emicha commented on etc/module.xml:
```diff
@@ -10,6 +10,6 @@
 
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-  <module name="SDM_Altapay" setup_version="3.2.1">
+  <module name="SDM_Altapay" setup_version="3.2.2">
```
Missing release notes

BushraAsif replied:
I have already updated the release notes which is recently merged.

emicha commented:
Why ins't the php sdk included as a composer dependency instead?

BushraAsif commented:
> Why ins't the php sdk included as a composer dependency instead?

Because when we install our plugin from Marketplace it does not fetch data from the composer so our code will break. Because its a marketplace requirement to have all code inside the repository.

BushraAsif commented:
> @BushraAsif the package.sh step should then do composer install before packaging, so all dependencies are stored in /vendor.
> We should not commit the PHP SDK classes inside the repository

@emicha We have updated the namespaces for the API files according to the marketplace and this will be a breaking change meaning that merchants need to completely remove our plugin and then reinstall it. We can make this change available in the future. We have updated the namespaces for the API files according to the marketplace and this will be a breaking change meaning that merchants need to completely remove our plugin and then reinstall it. We can make this change available in the future.
